<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>630</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Sweep Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>irmaker.png</normaloff>irmaker.png</iconset>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;aaaaaaaaaaaaaaaaaaaaaaaa&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QGroupBox" name="params_box">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Sweep parameters</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Beginning freq. (Hz)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ending freq. (Hz)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="beg_freq">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>35</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
    <property name="placeholderText">
     <string>in Hz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="end_freq">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>65</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>20000</string>
    </property>
    <property name="placeholderText">
     <string>in Hz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="duration">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>95</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <property name="placeholderText">
     <string>in sec</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration (sec)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="srate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>125</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="placeholderText">
     <string>in Hz</string>
    </property>
    <item>
     <property name="text">
      <string>44100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>48000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>88200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>176400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample rate (Hz)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>145</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Saving location</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>270</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Play sweep</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gen_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>270</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Generate sweep</string>
   </property>
  </widget>
  <widget class="PlotWidget" name="sweep_plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>600</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="PlotWidget" name="spectro_plot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>600</width>
     <height>220</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
